pkgs/by-name/un/unshieldv3/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "unshieldv3";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "wfr";
+    repo = "unshieldv3";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-InyWlSJ0u2PqeATb+pd6ELl3x9mhkmD/6gwiUFVc4ZA=";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/wfr/unshieldv3/commit/99a5cf5e04e67f37a58620343079b6c6a97e4eb3.patch";
+      hash = "sha256-yElDUxAchMLY784u4i7++YQvUlMGyDfhwCAi6PzExrI=";
+    })
```

Comment:
```suggestion
    # enable build on Clang
    (fetchpatch {
      url = "https://github.com/wfr/unshieldv3/commit/99a5cf5e04e67f37a58620343079b6c6a97e4eb3.patch";
      hash = "sha256-yElDUxAchMLY784u4i7++YQvUlMGyDfhwCAi6PzExrI=";
    })
```

Reply:
Oops, yep, that's a good idea. Added a comment.